### PR TITLE
product page and product page landscape

### PR DESCRIPTION
**Pull request by praveenreddy1798**
**Product Page**

Screenshot 2022-02-17 at 4 44 16 AM

**Product Page landscape**

Screenshot 2022-02-17 at 4 44 33 AM

### CODE REVIEW

**Review by TejasShekar — APPROVED** [review]
Clean code. 

Liked how you've used classes for your markup. Took a few pointers on how I should build my CL. Thankyou

File: containers/product-page-1/index.html [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="https://zingy-ui.netlify.app/main.css">
+    <link rel="stylesheet" href="../../utils.css">
+    <title>Zingy Deal | Product Page 1</title>
```
Comment (vaibhav18matere) [inline containers/product-page-1/index.html]: remove 1 just write `product page` or `product listing page`

File: containers/product-page-1/index.html [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+            <div class="flex align-center">
+                <h3 class="mg-r-xsm">Showing All Products</h3>
+                <h4 class="regular-dark">( Showing 20 Products )</h4>
+            </div>
+            <div class="flex-center mg-t-md gap-2">
+                <!-- Vertical Card -->
+                <div class='card bg-white'>
+                    <div class="card-image-container h-100 w-100 position-relative flex-evenly">
+                        <img class="card-image" src="../../assets/product.png" alt="product-image">
+                        <button class="btn btn-icon btn-icon-card position-absolute wishlist rounded flex-center">
```
Comment (vaibhav18matere) [inline containers/product-page-1/index.html:148]: refactor code, use prettier to make it aligned

**Review by vaibhav18matere — APPROVED** [review]
very clean code, most of the things are just perfect.
here is tiny improvement you can add :

File: containers/product-page-1/index.html (hunk 1 above)
Comment (praveenreddy1798) [reply containers/product-page-1/index.html]: Yes, will make this change

**Pull request merged**